src/Test/Utilities/Portable/Diagnostics/CommonDiagnosticAnalyzers.cs

```diff
@@ -572,7 +572,9 @@ public override void Initialize(AnalysisContext context)
         public sealed class AnalyzerWithCSharpCompilerDiagnosticId : DiagnosticAnalyzer
         {
             public static readonly DiagnosticDescriptor Descriptor = new DiagnosticDescriptor(
+#pragma warning disable RS1029 // Do not use reserved diagnostic IDs.
```

Comment:
Might want to turn off RS1029 for all non-shipping projects: https://github.com/dotnet/roslyn/blob/master/eng/config/rulesets/NonShipping.ruleset

Reply:
The targeted suppression seems fine for now, considering it was scope-limited